### ISSUE TITLE
Uuid-impl feature doesn't work

### ISSUE THREAD

**Issue by KPidS** [Aleph-Alpha/ts-rs#131]
I copied the [example](https://github.com/Aleph-Alpha/ts-rs/tree/main/example) and tried to `run` and `test` it, got this error: 
```
   Compiling example v0.1.0 (/home/kpids/Files/Code/ts-rs-tes)
error[E0277]: the trait bound `Uuid: TS` is not satisfied
  --> src/lib.rs:29:21
   |
29 | #[derive(Serialize, TS)]
   |                     ^^ the trait `TS` is not implemented for `Uuid`
   |
   = help: the following other types implement trait `TS`:
             &'static str
             ()
             (T1, T2, T3, T4, T5, T6, T7, T8, T9, T10)
             (T10,)
             (T2, T3, T4, T5, T6, T7, T8, T9, T10)
             (T3, T4, T5, T6, T7, T8, T9, T10)
             (T4, T5, T6, T7, T8, T9, T10)
             (T5, T6, T7, T8, T9, T10)
           and 51 others
   = note: this error originates in the derive macro `TS` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0277]: the trait bound `Uuid: TS` is not satisfied
   --> src/lib.rs:39:12
    |
39  |     token: Uuid,
    |            ^^^^ the trait `TS` is not implemented for `Uuid`
    |
    = help: the following other types implement trait `TS`:
              &'static str
              ()
              (T1, T2, T3, T4, T5, T6, T7, T8, T9, T10)
              (T10,)
              (T2, T3, T4, T5, T6, T7, T8, T9, T10)
              (T3, T4, T5, T6, T7, T8, T9, T10)
              (T4, T5, T6, T7, T8, T9, T10)
              (T5, T6, T7, T8, T9, T10)
            and 51 others
note: required by a bound in `Dependency::from_ty`
   --> /home/kpids/.cargo/registry/src/github.com-1ecc6299db9ec823/ts-rs-6.2.0/src/lib.rs:311:23
    |
311 |     pub fn from_ty<T: TS>() -> Option<Self> {
    |                       ^^ required by this bound in `Dependency::from_ty`

For more information about this error, try `rustc --explain E0277`.
error: could not compile `example` due to 2 previous errors
warning: build failed, waiting for other jobs to finish...
error: could not compile `example` due to 2 previous errors
```
Tried updating Rust with `rustup update`, tried different versions of `Uuid` and `ts-rs` crates, tried `cargo clean`, tried running it on stable and nightly. Still have no idea what causes it.

**Comment by timephy** [comment]
Hey, I am also interested in using `ts-rs` in combination with `uuid`.

Is there a plan for when said release is coming?

**Comment by fbucek** [comment]
I guess this problem is fixed with 7.0 release. I my case it works correctly with uuid version `1.4.1`
